okcupyd/json_search.py

```diff
@@ -26,7 +27,11 @@ def SearchFetchable(session=None, **kwargs):
 
     :param session: A logged in session.
     :type session: :class:`~okcupyd.session.Session`
+    :param order_by: Expected values: 'match', 'online', 'special_blend'
     """
+##:param location: A location string which will be used to filter results.
```

Comment:
Same as above. Also if this is kept, it needs to be indented properly. I've also used #: rather than ##: in this code base thus far.